recipes-core/glib-2.0/glib-2.0_2.70.3.bb

```diff
@@ -0,0 +1,54 @@
+require glib.inc
```

Comment:
Can we use glib from external meta-layers instead of copying the code here?

Reply:
It is from poky layer. current version in poky/meta is glib-2.0_2.62.6

Reply:
Can you please add a comment at least where does it come from?